### PR TITLE
EPMRPP-33466: React. Product status widget.

### PR DESCRIPTION
**Pull request by Andrew-Omelchenko**

### CODE REVIEW

File: app/src/components/widgets/tables/productStatus/itemLink/itemLink.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  render() {
+    const { value, tracking, customProps } = this.props;
+    return (
+      <div className={cx('main-info')}>
+        <NameLink
+          itemId={value.linkId}
+          ownLinkParams={customProps.ownLinkParams}
+          className={cx('name-link')}
+          onClick={() => tracking.trackEvent(LAUNCHES_PAGE_EVENTS.CLICK_ITEM_NAME)}
```
Comment (AmsterGet) [inline app/src/components/widgets/tables/productStatus/itemLink/itemLink.jsx]: Also not need to track event here

File: app/src/components/widgets/tables/productStatus/itemLink/itemLink.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    customProps: {
+      ownLinkParams: {},
+    },
+  };
+
+  render() {
+    const { value, tracking, customProps } = this.props;
+    return (
+      <div className={cx('main-info')}>
+        <NameLink
```
Comment (AmsterGet) [inline app/src/components/widgets/tables/productStatus/itemLink/itemLink.jsx]: I think not needed to create the separate component here. Just use `NameLink` component with necessary props inside the `NameColumn`

File: app/src/components/widgets/tables/productStatus/innerComponents.jsx [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+};
+
+export const PassingRateColumn = ({ className, value }, name, { formatMessage }) => {
+  if (value.type === FILTER_TITLE_TYPE) {
+    return null;
+  }
+  const passingRate = value.passingRate ? `${Math.round(value.passingRate)}%` : '';
+  return (
+    <div
+      className={cx(
```
Comment (AmsterGet) [inline app/src/components/widgets/tables/productStatus/innerComponents.jsx]:
```
className={cx('passing-rate-col', className, {
   'total-col': value.type === 'total',
    success: value.passingRate === 100,
    },
)}
```

File: app/src/components/widgets/tables/productStatus/innerComponents.jsx [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+        'passing-rate-col',
+        value.type === 'total' && 'total-col',
+        value.passingRate === 100 && 'success',
+        className,
+      )}
+    >
+      <div className={cx('desktop-block')}>{passingRate}</div>
+      <div className={cx('mobile-block', `passing-rate`)}>
+        <div className={cx('block-content', value.passingRate === 100 && 'success')}>
```
Comment (AmsterGet) [inline app/src/components/widgets/tables/productStatus/innerComponents.jsx]: `className={cx('block-content', { success: value.passingRate === 100 })}`

**Comment by Andrew-Omelchenko** [comment]
Fixed

File: app/src/components/widgets/tables/productStatus/innerComponents.jsx [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+    >
+      <div className={cx('main-info')}>
+        <NameLink itemId={value.linkId} ownLinkParams={ownLinkParams} className={cx('name-link')}>
+          <span title={value.name} className={cx('name')}>
+            {`${value.name ? formatItemName(value.name) : ''} `}
+          </span>
+        </NameLink>
+        <span className={cx('number-box')}>
+          <NameLink itemId={value.linkId} ownLinkParams={ownLinkParams} className={cx('name-link')}>
+            {value.number && <span className={cx('number')}>#{value.number}</span>}
```
Comment (AmsterGet) [inline app/src/components/widgets/tables/productStatus/innerComponents.jsx]: Which purpose here to render the `value.number` not near the [`value.name`](https://github.com/reportportal/service-ui/pull/1762/files#diff-30677d043f8af9b7050f24e0d006b5e6R66) in one `NameLink` component?

**Comment by Andrew-Omelchenko** [comment]
Fixed.

**Pull request merged**